test/powershell/Host/PSVersionTable.Tests.ps1

```diff
@@ -23,7 +23,7 @@ Describe "PSVersionTable" -Tags "CI" {
             $unexpectectGitCommitIdPattern = $fullVersionPattern
         }
 
-        $powerShellVersions = "1.0", "2.0", "3.0", "4.0", "5.0", "5.1", "6.0", "6.1", "6.2", "7.0", "7.1", "7.2", "7.3"
+        $powerShellVersions = "3.0", "4.0", "5.0", "5.1", "6.0", "6.1", "6.2", "7.0", "7.1", "7.2", "7.3"
```

Comment:
Maybe also remove 6.0/1/2, given that they have reached end of life.

Reply:
Great

Reply:
> Maybe also remove 6.0/1/2, given that they have reached end of life.

I wonder how are the compatibility values used by developers?